ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__multi-stable-openshift-logging-6.2.yaml

```diff
@@ -118,6 +118,74 @@ tests:
     - ref: install-operators
     - chain: openshift-e2e-test-logging-qe
     workflow: cucushift-installer-rehearse-azure-ipi
+- as: baremetal-sno-agent-ipv4-f999-logging
+  cluster: build05
+  cron: 8 8 29 2 *
```

Comment:
Seems this job only can be triggered on Feb 29th.

Reply:
I don't want it to be triggered automatically.  So we use Feb 29th here.